Views/Todo/Update.cshtml

```diff
@@ -0,0 +1,118 @@
+﻿@model Efcore_demo.Models.TodoDto
+
+@{
+    ViewBag.Title = "Edit Todo";
+    Layout = "_Layout";
+}
+
+<div class="container mt-5">
+    <div class="form-container">
+        <h2 class="form-title">Edit Todo</h2>
+        <form method="post" class="todo-form">
+            <input type="hidden" asp-for="Id" />
+
+            <div class="form-group">
+                <label for="TaskTitle">Title</label>
+                <input type="text" asp-for="TaskTitle" class="form-control" placeholder="Enter the task title" />
+            </div>
+
+            <div class="form-group">
+                <label for="TaskDescription">Description</label>
+                <textarea asp-for="TaskDescription" class="form-control" rows="3" placeholder="Enter a brief description"></textarea>
+            </div>
+
+            <div>
+                <input type="hidden" asp-for="TaskDate" />
```

Comment:
Why you are hiding TaskDate

Reply:
taskdate is auto generated as task is created 
taskdate= datetime.now();